hack/lib/util/logs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+get_logging_pod_logs() {
+  local node
+  for node in $(oc get nodes --no-headers | awk '{print $1}') ; do
+      oc adm node-logs $node --path=fluentd/fluentd.log > $ARTIFACT_DIR/$node-fluentd.log
```

Comment:
hmm - is there some way to use `oc adm node-logs` to grab `fluentd/fluent.*.log` too?  If there is a lot of output, fluentd will do log rotation, and you will want to grab all of them, rotated logs too.

Reply:
Let me give it a try; I think it's possible